test/src/main/java/org/axonframework/test/aggregate/AggregateTestFixture.java

```diff
@@ -506,16 +510,28 @@ private void registerAggregateCommandHandlers() {
     }
 
     private void ensureRepositoryConfiguration() {
-        if (repository == null) {
+        if(repository != null) {
+            return;
+        }
+
+        if(this.useStateStorage) {
+            this.registerRepository(new InMemoryRepository<>(
+                    aggregateType,
+                    subtypes,
+                    eventStore,
+                    getParameterResolverFactory(),
+                    getHandlerDefinition(),
+                    getRepositoryProvider()));
+        } else {
             AggregateModel<T> aggregateModel = aggregateModel();
-            registerRepository(EventSourcingRepository.builder(aggregateType)
-                                                      .aggregateModel(aggregateModel)
-                                                      .aggregateFactory(new GenericAggregateFactory<>(aggregateModel))
-                                                      .eventStore(eventStore)
-                                                      .parameterResolverFactory(getParameterResolverFactory())
-                                                      .handlerDefinition(getHandlerDefinition())
-                                                      .repositoryProvider(getRepositoryProvider())
-                                                      .build());
+            this.registerRepository(EventSourcingRepository.builder(aggregateType)
+                    .aggregateModel(aggregateModel)
+                    .aggregateFactory(new GenericAggregateFactory<>(aggregateModel))
+                    .eventStore(eventStore)
+                    .parameterResolverFactory(getParameterResolverFactory())
+                    .handlerDefinition(getHandlerDefinition())
+                    .repositoryProvider(getRepositoryProvider())
+                    .build());
```

Comment:
The indentation of the builder invocations is actually intended to be where it used to be.
Might be something to do with the selected style guide. You can select Axon's style guide for your PR if you want, by using [this](https://github.com/AxonFramework/AxonFramework/blob/master/axon_code_style.xml) file.
Simply indenting it by hand is obviously also fine.

Reply:
Thanks for pointing me to the configuration.